jsonargparse_tests/test_dataclass_like.py

```diff
@@ -978,3 +984,17 @@ def test_field_init_false(self, parser):
         assert cfg == Namespace()
         init = parser.instantiate_classes(cfg)
         assert init.data.p1 == {}
+
+    def test_nested_dataclass_with_default(self, parser):
```

Comment:
```suggestion
    def test_nested_with_default(self, parser):
```
For consistency in naming and because in the same file there are tests for python dataclasses which are different to the attrs ones.